packages/mds-types/index.ts

```diff
@@ -51,22 +51,26 @@ export type VEHICLE_STATE = typeof VEHICLE_STATES[number]
 // export const RIGHT_OF_WAY_STATUSES = ['available', 'reserved', 'unavailable', 'trip']
 export const RIGHT_OF_WAY_STATES = ['available', 'reserved', 'non_operational', 'trip'] as const
 
-// export const VEHICLE_EVENTS = Enum(
-//   'register',
-//   'service_start',
-//   'service_end',
-//   'provider_drop_off',
-//   'provider_pick_up',
-//   'agency_pick_up',
-//   'agency_drop_off',
-//   'reserve',
-//   'cancel_reservation',
-//   'trip_start',
-//   'trip_enter',
-//   'trip_leave',
-//   'trip_end',
-//   'deregister'
-// )
+export const VEHICLE_EVENTS_0_4_1 = [
```

Comment:
Should probably add a comment above this `const` (and the corresponding `type` definition that follow) with an `@deprecated` tag, and explanation about what this is used for

Reply:
I don't think we can really say it's deprecated until we stop supporting 0.4.1, but a comment's a good idea.